File: Exercicio 9/Exercicio 9.cpp
#include<stdio.h>
#include <stdlib.h>

main()
{
	int n = 6,i = 3, j = 2, NumerosMultiplos = 0;
	
	printf("Insira o valor de N: ");
	scanf("%d", &n);
	
	printf("Insira o valor de i: ");
	scanf("%d", &i);
	
	printf("Insira o valor de j: ");
	scanf("%d", &j);
	
	for(int Num = 0; NumerosMultiplos < n; Num++)
	{
		if(Num%i == 0)
		{
			printf("%d, ", Num);
			NumerosMultiplos++;
		}
		else if(Num%j == 0)
		{
			printf("%d, ", Num);
			NumerosMultiplos++;
		}
	}
	
	
    system("pause");
}

File: PHP/Exercicio 9/exercicio9.php
<?php
$n = intval($_GET["n"]);
$i = intval($_GET["i"]);
$j = intval($_GET["j"]);

$numerosMultiplos = 0;

for($num = 0; $numerosMultiplos < $n; $num++){
  if($num % $i == 0){
    echo "$num,";
    $numerosMultiplos++;
  }else if($num % $j == 0){
    echo "$num,";
    $numerosMultiplos++;
  }
}
?>
